### ISSUE TITLE
QuickCharge/Rockets Crossbow

### ISSUE THREAD

**Issue by godplayerlennyTV** [HGLabor/uhc#13]
Das Entchantement QuickCharge ist mit Piercing auf dem Crossbow ziemlich Overpowered und mit Rockets dazu viel zu krank. Wird sehr warscheinlich noch zum Problem werden.

**Comment by Indikativ** [comment]
klar wie soll man auch rockets innerhalb 30 minuten holen außerdem machen die nicht viel schaden wenn man nicht gerade 3 stacks gunpowder übrig hat, die man ja von creepern bekommt, welche ich im letzten  game nicht mal einen gesehen habe

**Comment by godplayerlennyTV** [comment]
Jo, sorry falls ich dir irgendwie was getan habe, aber Rockets machen schon gut damage und wenn man lucky ist und gut Creeper findet ist schon ziemlich stark, vielleicht nicht instant abwerten weil du in einem Game mal kein Creeper gesehen hast... :)

**Comment by DoctorFuchs** [comment]
> Jo, sorry falls ich dir irgendwie was getan habe, aber Rockets machen schon gut damage und wenn man lucky ist und gut Creeper findet ist schon ziemlich stark, vielleicht nicht instant abwerten weil du in einem Game mal kein Creeper gesehen hast... :)

Indikativ ist oft ziemlich abwertend, mach dir nichts draus

**Comment by Indikativ** [comment]
das stimmt glaube ich nicht ich hab dir ja nur gesagt dass man in der 1.16 sich keine schneebälle oder eier macht

**Comment by DoctorFuchs** [comment]
@Indikativ nicht schlimm, es ist gut kritik zu üben und anderen Tipps zu geben. (Das ist ja auch das Prinzip von Github) Aber manchmal vielleicht mit mehr Erklärung. (Der Version oder hier der Modifizierungen in der Spawnrate)

@godplayerlennyTV Ansonsten gute Idee, die vielleicht in diesem Spielmodus wenig Sinn macht.
